apps/web/src/views/Ifos/components/IfoFoldableCard/IfoPoolCard/ContributeModal.tsx

```diff
@@ -40,9 +41,6 @@ interface Props {
 
 const multiplierValues = [0.1, 0.25, 0.5, 0.75, 1]
 
-// Default value for transaction setting, tweak based on BSC network congestion.
```

Comment:
if this is needed for some config or default setting could we add to a constants file somewhere. maybe make a map of Default gas settings like 

```ts
export const DefaultGasPrices = {
   Slow: parseEther('5', 'gwei'),
   Medium: parseEther('10', 'gwei'),
   Fast: parseEther('15', 'gwei')
}


const gasPrice = DefaultGasPrices.Medium

```

Reply:
Yeah we can consider a default map but based on chain as well. 

Right now while calling we're relying on `callWithGasPrice` which should estimate gas with some buffer already. perhaps when this was coded initially this wasn't there 🤔

Reply:
ahh ok i see. thanks for the clarity